### PR TITLE
CNDIT-1568: Update the liquibase module

### PR DESCRIPTION
**Pull request by alog-enquizit**
- Update the docker file to include all the sql scripts in the image. 
- Update the Build-and-deploy-reporting-services.yaml

### CODE REVIEW

File: person-service/src/main/java/gov/cdc/etldatapipeline/person/transformer/DataPostProcessor.java [hunk 1]
```diff
@@ -82,9 +82,11 @@ public <T extends PersonExtendedProps> void processPersonRace(String race, T pf)
     public <T extends PersonExtendedProps> void processPersonTelephone(String telephone, T pf) {
         if (!ObjectUtils.isEmpty(telephone)) {
             Function<String, T> personPhoneFn =
-                    (useCd) -> Arrays.stream(utilHelper.deserializePayload(telephone, Phone[].class))
-                            .filter(phone -> StringUtils.hasText(phone.getUseCd())
-                                    && phone.getUseCd().equalsIgnoreCase(useCd))
+                    (code) -> Arrays.stream(utilHelper.deserializePayload(telephone, Phone[].class))
+                            .filter(phone -> (StringUtils.hasText(phone.getUseCd())
+                                    && phone.getUseCd().equalsIgnoreCase(code)) ||
+                                    (StringUtils.hasText(phone.getCd())
```
Comment (alog-enquizit) [inline person-service/src/main/java/gov/cdc/etldatapipeline/person/transformer/DataPostProcessor.java:88]: @upasanapattnaik-eq Did you already port in this changes in your pR?

Comment (upasanapattnaik-eq) [reply person-service/src/main/java/gov/cdc/etldatapipeline/person/transformer/DataPostProcessor.java:88]: Yup! They should be in a PR in the other repo.

File: liquibase-service/src/main/resources/db/changelog/db.odse.changelog-15.1.yaml [hunk 2]
```diff
@@ -145,12 +145,12 @@ databaseChangeLog:
       author: liquibase
       changes:
         - sqlFile:
-            path: src/main/resources/db/odse/020-v_inv_form_code_data-001.sql
+            path: src/main/resources/db/odse/019-v_inv_form_code_data-001.sql
```
Comment (upasanapattnaik-eq) [inline liquibase-service/src/main/resources/db/changelog/db.odse.changelog-15.1.yaml:148]: I think this is 020 in the main repo. Are there any changes to the order?

File: liquibase-service/src/main/resources/db/rdb/005-create_nrt_provider_key.sql [hunk 3]
```diff
@@ -1,4 +1,4 @@
-use RDB_MODERN;
+
```
Comment (upasanapattnaik-eq) [inline liquibase-service/src/main/resources/db/rdb/005-create_nrt_provider_key.sql:1]: Thank you so much for this update!!

**Review by upasanapattnaik-eq — APPROVED** [review]
LGTM!

File: liquibase-service/src/main/resources/db/changelog/db.odse.changelog-15.1.yaml (hunk 2 above)
Comment (alog-enquizit) [reply liquibase-service/src/main/resources/db/changelog/db.odse.changelog-15.1.yaml:148]: @upasanapattnaik-eq I had the sql-server agent override over here as `019`. It should be run against `master` 
Hence the reordering.

**Pull request merged**